### DISCUSSION TITLE
What is "self"?

### DISCUSSION THREAD

**Discussion by miasik** [hako-mikan/sd-webui-supermerger#165] [Q&A]
I see a new mode "self" in the "Calculation mode" but can't find it in the documentation. What is it?

**Comment by CapsAdmin** [comment]
According to source code it multiplies Model A by alpha.

https://github.com/hako-mikan/sd-webui-supermerger/blob/main/scripts/mergers/mergers.py#L567

Not sure what the use case is really.

**Comment by hako-mikan** [comment]
As he says, it is a feature to significantly increase or decrease weights for the original model. This usually does not work well with regular merging. Function for block or Elemental merge. With block  merging, I expect to manipulate various elements for the model, but currently, there is not enough validation. I think increasing or decreasing specific elements will also affect the brush strokes.

**Reply by miasik** [reply]
The both explanations are clear. I can change some weights in a model for future merging. Am I right?

in theory it is simple.
First, I increase (for example) some weights in model A. 
Then I merge A with B having more of A in the result because of increased weights.

Could anyone please give a real example/case of using this technique?

**Reply by daileta1** [reply]
I have one I'm experimenting with...

When mixing a model with a particular style I want onto another model with some type of conceptual understanding, I'll over merge the style model using add difference at a strength of 1 to 2. For many models, less than 1 results in a merged style, but over 1 starts to look oversaturated. Once the model is merged, then do a self merge with an inverse ratio alpha at increasing values until the model stabilizes.

**Reply by miasik** [reply]
Looks like a cure for oversaturated/burned models. I should check it on my own.
Thanks!

**Reply by daileta1** [reply]
Try it with mbw to see if it'll work in your case. I don't know if the alpha in that calcmode will also affect the base, but decreasing the weights on the base may or may not have desirable effects.
